internal/app/store/store_test.go

```diff
@@ -0,0 +1,210 @@
+package store_test
+
+import (
+	"errors"
+	"github.com/gasparian/money-transfers-api/internal/app/models"
+	"github.com/gasparian/money-transfers-api/internal/app/store"
+	"github.com/gasparian/money-transfers-api/internal/app/store/kvstore"
+	"github.com/gasparian/money-transfers-api/internal/app/store/sqlstore"
+	"os"
+	"testing"
+)
+
+var (
+	invalidBalanceValueErr  = errors.New("Invalid balance value")
+	transactionCorruptedErr = errors.New("Transaction corrupted")
+)
+
+func testStore(store store.Store, t *testing.T) {
+	t.Run("InsertAccount", func(t *testing.T) {
+		balance := models.MoneyAmount{Integer: 10, Fraction: 5}
+		acc, err := store.InsertAccount(balance)
+		if err != nil {
+			t.Fatal(err)
+		}
+		if models.CompareMoney(&balance, &acc.Balance) != 0 {
```

Comment:
In most of the cases, you use `models.CompareMoney(&balance, &acc.Balance)` as equals or not equals function. You can use the `reflect/deepequal.go` from the standard library to solve this problem - https://golang.org/pkg/reflect/#DeepEqual